libraries/bezier-rs/src/bezier/lookup.rs

```diff
@@ -45,28 +22,54 @@ impl Bezier {
 				high = mid;
 			}
 		}
-		self.unrestricted_parametric_evaluate(mid)
+
+		mid
 	}
 
-	/// Calculate the point on the curve based on the `t`-value provided.
-	/// Expects `t` to be within the inclusive range `[0, 1]`.
-	pub fn evaluate(&self, t: ComputeType) -> DVec2 {
-		match t {
+	/// Convert a [ComputeType] to a parametric `t`-value.
+	pub(crate) fn compute_type_to_parametric(&self, c: ComputeType) -> f64 {
```

Comment:
Rename `c` to `compute_type` since we avoid cryptic variable names.

Reply:
Our library and Bezier.js always use the argument `t` to represent specifically the parametric case, so I'm not fond of using `t` or `ComputeT` since `ComputeType` has the Euclidean variant which means something very different. I've added documentation now to the ComputeType enum which I hope makes the distinction more clear.

In general, I don't love the name `ComputeType` either, but we called it this because Parametric and Euclidean are each methods to **compute** a point along the curve. We could go a different route and name the Enum / instances something like `PointRef`  / `p` since the data essentially references a point on the curve, if you like that better, or anything you had in mind.

Reply:
I've renamed this Enum to `TValue`. This has the unfortunate side effect of bloating the MR somewhat, as many variables in the frontend and wasm layers had to be renamed appropriately.